SU2_CFD/include/drivers/CDiscAdjMultizoneDriver.hpp

```diff
@@ -141,6 +141,16 @@ class CDiscAdjMultizoneDriver : public CMultizoneDriver {
    */
   void StartSolver() override;
 
+  /*!
+   * \brief [Overload] Launch the tape test mode for the discrete adjoint multizone solver.
+   */
+  void TapeTest();
+
+  /*!
+   * \brief [Overload] Get error numbers after a tape test run of the discrete adjoint multizone solver.
+   */
+  int TapeTest_GatherErrors(struct AD::ErrorReport& error_report);
```

Comment:
Can all these things be const? You don't need to add struct to function arguments (I didn't even know this was allowed).
```suggestion
  int TapeTestGatherErrors(const AD::ErrorReport& error_report) const;
```

Reply:
Everything that can be const, should be const.